ansible/roles/common/tasks/main.yml

```diff
@@ -23,3 +23,40 @@
   apt: name="libssl1.0.0" state=present
   tags:
     - upgrade_openssl
+
+
+- name: Hard file limits
+  sudo: yes
+  pam_limits:
```

Comment:
This PR should have done that already: https://github.com/dimagi/commcarehq-ansible/pull/617 (specifically commit 2e88b685)